src/libutil/error.cc

```diff
@@ -262,6 +262,28 @@ std::ostream & showErrorInfo(std::ostream & out, const ErrorInfo & einfo, bool s
         prefix += ":" ANSI_NORMAL " ";
 
     std::ostringstream oss;
+
+    // traces
+    if (showTrace && !einfo.traces.empty()) {
+        for (auto iter = einfo.traces.begin(); iter != einfo.traces.end(); ++iter) {
```

Comment:
does anything speak against:

```suggestion
        for (const auto &trace : traces) { 
        ...
```

?

Reply:
We usually put the spaces on both sides of the `&`
```suggestion
        for (const auto & trace : traces) { 
```

Reply:
I added that change as new commit - thanks a lot!